### PR TITLE
[bg #159243208] Remove for-loops from code

### PR DESCRIPTION
**Pull request by uchemukolo**
#### What does this PR do?
- remove for-loop to comply with ES6 standard
- Use The findIndex() method which returns the index of the first element in the array that satisfies the provided testing function.
#### Description of Task to be completed?
- remove for-loop to comply with ES6 standard
- Use The findIndex() method which returns the index of the first element in the array that satisfies the provided testing function.
#### How should this be manually tested?
npm run test
#### Any background context you want to provide?
- javaScript for-loop was used to implement this before
#### What are the relevant pivotal tracker stories?
[#159243208](<https://www.pivotaltracker.com/story/show/159243208>)
#### Screenshots (if appropriate)
N/A
#### Questions:

### CODE REVIEW

File: server/controllers/entries.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import data from '../models/data';
+import { runInDebugContext } from 'vm';
```
Comment (fidelisojeah) [inline server/controllers/entries.js]: Why is this here?

File: server/controllers/entries.js [hunk 2]
```diff
@@ -37,21 +38,21 @@ class Entry {
...
-    for (let i = 0; i < entry.length; i++) {
-      if (entry[i].entryId === parseInt(request.params.entryId, 10)) {
-        return response.send({
-          message: 'Successful',
-          entry: data[i],
-          error: false
-        });
-      }
+static getOne(request, response) {
+  let index = entry.findIndex(item => item.entryId === parseInt(request.params.entryId, 10)); {
```
Comment (fidelisojeah) [inline server/controllers/entries.js]: Good job (y) Though if the value of index is not going to change, use `const` rather than `let`

**Review by fidelisojeah — CHANGES_REQUESTED** [review]

File: server/controllers/entries.js (hunk 1 above)
Comment (uchemukolo) [reply server/controllers/entries.js]: i think i clicked it by mistake. will rectify now

File: server/controllers/entries.js (hunk 2 above)
Comment (uchemukolo) [reply server/controllers/entries.js]: ok, will do.
